test/js_optimizer/test-object-literals.js

```diff
@@ -0,0 +1,14 @@
+var foo = 1;
+var baz = {};
+
+// Verify that shorthand object literals are preserved.
+var bar = {
+  foo,
+  baz,
+};
+
+// Objects that don't use shorthand should also be preserved, as-is
```

Comment:
This looks modified in the output above? `foo: foo,` was turned into just `foo,`.

But that seems ok - is there a reason to avoid that?

Reply:
Oh.. I thought it was able to distinguish.  I guess not.  I'll update the comment.